wazo_calld/helpers/ari_.py

```diff
@@ -141,11 +141,19 @@ def is_local(self):
         return channel.json['name'].startswith('Local/')
 
     def is_caller(self):
+        try:
+            user_outgoing_call = self._get_var('WAZO_USER_OUTGOING_CALL')
+            return user_outgoing_call == 'true'
+        except ARINotFound:
+            pass
+
```

Comment:
Why we cannot set and use WAZO_CHANNEL_DIRECTION?

Reply:
The outgoing call endpoint is not called from the dialplan. It replaces the endpoint's context.

Reply:
Ok but why not setting the WAZO_CHANNEL_DIRECTION in the same place that https://github.com/wazo-pbx/wazo-calld/pull/16/files#diff-e84777736f838bad7bd76cab95d7d04eR76

Reply:
Ok it doesn't make sense to use WAZO_CHANNEL_DIRECTION